dagger/ci/main.go

```diff
@@ -27,6 +27,14 @@ func (m *Ci) Test(
 
 	return m.codeContainer(source, testTargetNetwork, recallPrivateKey).
 		WithServiceBinding("localnet", m.localnetService(dockerUsername, dockerPassword)).
+		WithExec([]string{
+			"sh", "-c",
+			"make test",
+		}).
+		WithExec([]string{
+			"sh", "-c",
+			"recall --network localnet account deposit 1",
```

Comment:
These can include any CLI tests we want. Can move the actual commands out to other scripts that can be invoked from here in subsequent PRs.